decidim-core/app/packs/stylesheets/decidim/_login.scss

```diff
@@ -12,18 +17,50 @@
   }
 
   &__omniauth {
-    @apply flex flex-col items-center gap-8 text-gray-2 font-semibold w-fit mx-auto mt-10;
+    @apply flex flex-row items-center gap-8 text-gray-2 font-semibold w-fit mx-auto mt-10 [&>a]:w-auto;
 
     &-button {
-      @apply button button__xl border border-gray shadow-[0_4px_6px_rgba(211,211,211,0.25)] justify-start gap-8 text-lg w-full;
+      @apply button__lg border border-gray shadow-[0_4px_6px_rgba(211,211,211,0.25)] justify-start gap-8 text-lg w-full rounded;
 
       svg {
-        @apply w-6 h-6 fill-current flex-none;
+        @apply w-6 h-6 flex-none;
+      }
+
+      span {
+        @apply hidden xl:inline;
+      }
+
+      &--developer {
```

Comment:
I think we should add to the release notes some kind of information about how to change the CSS associated with : `login__omniauth-button--%{provider}`

Reply:
See commit: https://github.com/decidim/decidim/pull/13481/commits/93d04f848daff79fec693ef35b03cd6127373158